client/modules/IDE/components/ConsoleInput.jsx

```diff
@@ -117,7 +153,6 @@ function ConsoleInput({ theme, dispatchConsoleEvent, fontSize }) {
 
 ConsoleInput.propTypes = {
   theme: PropTypes.string.isRequired,
-  dispatchConsoleEvent: PropTypes.func.isRequired,
   fontSize: PropTypes.number.isRequired
 };
```

Comment:
`theme` and `fontSize` are both selected from Redux.  so those can be selected inside the `ConsoleInput` rather than selected by the parent and passed down.

Reply:
Although now I see that the parent still needs to select them regardless because it uses them for the `ConsoleFeed` style.